### PR TITLE
Add sec/seo step to autoaccept

### PR DESCRIPTION
**Pull request by ghost**
Add new table `DomainAcceptance` which is in relation with `Domain`.
This is backward compatible change because SEC/SEO defaults are False this.

### CODE REVIEW

File: powerdns/models/powerdns.py [hunk 1]
```diff
@@ -668,6 +681,25 @@ def __str__(self):
         return str(self.domain)
 
 
+class DomainAcceptance(models.Model):
+    domain = models.OneToOneField(
```
Comment (mkurek) [inline powerdns/models/powerdns.py]: why couldn't `require_sec_acceptance` and `require_seo_acceptance` be attribute of `Domain`?

File: powerdns/models/requests.py [hunk 2]
```diff
@@ -131,6 +136,22 @@ def reject(self):
...
 
+    def is_seo_acceptance_required(self):
+        """
+        Check if delete request requires SEO acceptance.
+        """
+        if self.owner and self.owner.is_superuser:
+            return False
+        return (
+            isinstance(self.target, Record) and
+            self.target.type in {'A', 'AAAA', 'CNAME'} and
```
Comment (mkurek) [inline powerdns/models/requests.py]: i'd add record types to settings (both for seo and sec acceptance)

Comment (ghost) [reply powerdns/models/requests.py]: You mean make this `{'A', 'AAAA', 'CNAME'}` set configurable by settings?

File: powerdns/models/requests.py [hunk 3]
```diff
@@ -446,5 +467,20 @@ def as_history_dump(self):
             'type':  self.target_type or '',
         }
 
+    def is_sec_acceptance_required(self):
```
Comment (mkurek) [inline powerdns/models/requests.py:477]: could you make some tests for this method?

File: powerdns/models/requests.py [hunk 4]
```diff
@@ -131,6 +136,22 @@ def reject(self):
     def __str__(self):
         return 'Delete {}'.format(self.target)
 
+    def is_seo_acceptance_required(self):
```
Comment (mkurek) [inline powerdns/models/requests.py:142]: could you make some tests for this method?

**Review by mkurek — CHANGES_REQUESTED** [review]

**Review by mkurek — APPROVED** [review]

**Review by szok — APPROVED** [review]

**Pull request merged**